content/docs/intro/languages/dotnet.md

```diff
@@ -0,0 +1,94 @@
+---
+title: ".NET"
+menu:
+  intro:
+    parent: languages
+    weight: 2
+
+aliases: ["/docs/reference/dotnet/"]
+---
+
+{{< langchoose dotnetonly >}}
```

Comment:
Not sure this is needed on this page.  We could just have details for each language all inline in the content.  Hiding the information about other .NET languages probably hurts more than helps in this context.

Reply:
accepted